### PR TITLE
CR-1050004 and CR-1048118

### PR DESCRIPTION
**Pull request by AShivangi**
- CR-1050004
bogus cards up to date message during update when there are multiple platforms on the system
- CR-1048118
xbutil list reports card not ready and defers to xbmgmt flash --scan --verbose which does not give more info to the problem

### CODE REVIEW

File: src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp [hunk 1]
```diff
@@ -110,12 +110,7 @@ static void print_pci_info(std::ostream &ostr)
     if (pcidev::get_dev_total() != pcidev::get_dev_ready()) {
         ostr << "WARNING: "
             << "card(s) marked by '*' are not ready, is MPD runing? "
-            << "run 'systemctl status mpd' to check MPD details.";
-	    if (pcidev::get_dev_total(false) == 0)
-            ostr << std::endl;
-	    else
-            ostr << " please also run 'xbmgmt flash --scan --verbose' to further check card details."
-                << std::endl;
+            << "run 'systemctl status mpd' to check MPD details." << std::endl;
```
Comment (maxzhen) [inline src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp:113]: let's just say that card is not ready, remove all guess work.

Comment (AShivangi) [reply src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp]: Done

**Comment by maxzhen** [comment]
retest this please

**Comment by abalasa** [comment]
retest this please

**Comment by maxzhen** [comment]
Please separate multiple CR numbers with ‘,’

**Comment by abalasa** [comment]
retest this please

**Comment by AShivangi** [comment]
retest this please

**Comment by AShivangi** [comment]
retest this please

**Pull request merged**